src/stylo/shadow_cljs.clj

```diff
@@ -0,0 +1,65 @@
+(ns stylo.shadow-cljs
+  (:require [clojure.java.io :as io]
+            [shadow.jvm-log :as log]
+            [stylo.core]))
+
+(def css-path-key ::css-path)
+
+
+(defn get-path [build-state]
+  (get build-state css-path-key))
+
+
+(defn assoc-path [build-state path]
+  (assoc build-state css-path-key path))
```

Comment:
I think this code is redundant

Reply:
It is true only for 
```clojure
(defn get-path [build-state]
   (get build-state css-path-key))
```

`assoc-path` is used to save the file name in build-state with `css-path-key` key. That can be used to update the page template's link element or append other styles to the same file or something.